templates/openy-repeat-schedule-dashboard--sidebar.html.twig

```diff
@@ -135,8 +135,8 @@
           <div v-if="getInstructorFilter()" class="form-group-wrapper">
             <label for="form-group-instructor-name" data-toggle="collapse" data-target="#form-group-instructor-name" role="button" class="expanded collapsed" aria-controls="form-group-instructor-name" aria-expanded="false">
               {{ 'Instructor name'|t }}
-              <span class="material-symbols-outlined icon minus">add_circle</span>
-              <span class="material-symbols-outlined icon plus">do_not_disturb_on</span>
+              <span class="material-symbols-outlined icon plus">add_circle</span>
+              <span class="material-symbols-outlined icon minus">do_not_disturb_on</span>
```

Comment:
Why do we use "_" instead of whitespace? Is there any limitation?

Reply:
@podarok 
This is a way the Google Font Icons library [works](https://fonts.google.com/icons?icon.size=48&icon.color=%235f6368&icon.style=Sharp&icon.query=cards&selected=Material+Symbols+Sharp:space_dashboard:FILL@0;wght@400;GRAD@0;opsz@48) - 
![image](https://github.com/user-attachments/assets/bdcd1f04-108c-4e2b-8a77-83324b7d518d)
Or do I misunderstand what you mean

Reply:
got it, disregard